x-pack/plugins/ml/public/application/services/annotations_service.tsx

```diff
@@ -79,3 +79,25 @@ export const annotation$ = new BehaviorSubject<AnnotationState>(null);
 */
 export const annotationsRefresh$ = new BehaviorSubject(Date.now());
 export const annotationsRefreshed = () => annotationsRefresh$.next(Date.now());
+
+export class AnnotationUpdatesService {
+  private _annotation$: BehaviorSubject<AnnotationState> = new BehaviorSubject<AnnotationState>(
+    null
+  );
+
+  public get() {
+    return this._annotation$;
+  }
```

Comment:
you shouldn't expose a subject instance, `update$` is already taking care of updates. 
```suggestion
```

Reply:
Updated here https://github.com/elastic/kibana/pull/83216/commits/0cf7ad04d7cb582f445be2eca121a0f5946ab03e